content/kubernetes/deployment/openshift/openshift-cli.md

```diff
@@ -22,317 +22,230 @@ aliases: [
+1. Patch the validating webhook with the certificate. 
 
     ```sh
-    oc apply -f your_cluster_name.yaml
+    kubectl patch ValidatingWebhookConfiguration redb-admission --patch "$(cat modified-webhook.yaml)"
     ```
 
-1. Run `oc get rec` and verify that creation succeeded. (`rec` is a shortcut for “RedisEnterpriseClusters”).
+### Limit the webhook to relevant namespaces
 
-    The command returns a confirmation response such as:
+If not limited, the webhook will intercept requests from all namespaces. If you have several REC objects in your Kubernetes cluster, you need to limit the webhook to the relevant namespaces. If you aren't using multiple namespaces, you can skip this step.
 
-    ```sh
-    NAME AGE
+1. View your namespace YAML file to verify your namespace is labeled and the label is unique to this namespace (see example below).
 
-    Your_cluster_name 17s
+    ```bash
+    apiVersion: v1
+    kind: Namespace
+    metadata:
+      labels:
+        namespace-name: staging
+    name: staging
     ```
 
-Your Cluster will be ready shortly, typically within a few minutes.
+1. Patch the webhook spec with the `namespaceSelector` field. 
+    ```bash
+    cat > modified-webhook.yaml <<EOF
+    webhooks:
+    - name: redb.admission.redislabs
+      namespaceSelector:
+        matchLabels:
+          namespace-name: staging
+    EOF
 
-To check the cluster status, run:
+    ```
 
-```sh
-kubectl get pod
-```
+1. Apply the patch.
 
-You should receive a response similar to the following:
+    ```bash
+    kubectl patch ValidatingWebhookConfiguration redb-admission --patch "$(cat modified-webhook.yaml)"
+    ```
 
-| NAME                             | READY | STATUS  | RESTARTS | AGE |
-| -------------------------------- | ----- | ------- | -------- | --- |
-| your_cluster_name-0              | 2/2   | Running | 0        | 1m  |
-| your_cluster_name-1              | 2/2   | Running | 0        | 1m  |
-| your_cluster_name-2              | 2/2   | Running | 0        | 1m  |
-| your_cluster_name-controller-x-x | 1/1   | Running | 0        | 1m  |
-| Redis-enterprise-operator-x-x    | 1/1   | Running | 0        | 5m  |
+### Verify the admission controller installation
 
-Next, create your databases.
+1. Apply an invalid resource (provided below).
 
-## Step 5: Create a database
+    This should force the admission controller to reject it. If it applies successfully, the admission controller is not installed correctly.
 
-To create your database:
+      ```bash
+      $ kubectl apply -f - << EOF
+      apiVersion: app.redislabs.com/v1alpha1
+      kind: RedisEnterpriseDatabase
+      metadata:
+        name: redis-enterprise-database
+      spec:
+       evictionPolicy: illegal
+      EOF
+      ```
 
-1. Apply port forwarding to your Cluster:
 
-    ```sh
-    oc port-forward your_cluster_name-0 8443:8443
+    You should see an error that the admission controller webhook `redb.admission.redislabs`.
```

Comment:
Looks like this sentence is missing something.

Reply:
I meant "from" not "that". Good catch!